bagit.xml

```diff
@@ -1,819 +1,800 @@
-</t>
-
-<figure>
- <artwork>
+  </artwork>
+          </figure>
+          <t>
+            Each line of a payload manifest file &must; be of the form:
+          </t>
+          <figure>
+            <artwork>checksum filename</artwork>
+            <postamble>
+    where <spanx style="emph">filename</spanx> is the pathname of a file
+    relative to the base directory, and <spanx style="emph">checksum</spanx> is
+    a hex-encoded checksum calculated according to
+    <spanx style="emph">algorithm</spanx> over every octet in the file.
+  </postamble>
+          </figure>
+          <t>
+            <list style="symbols">
+              <t>The hex-encoded checksum &may; use uppercase and/or lowercase letters.</t>
+              <t>The slash character ('/') &must; be used as a path separator
+                in <spanx style="emph">filename</spanx>.</t>
+              <t>One or more linear whitespace characters (spaces or tabs)
+                &must; separate <spanx style="emph">checksum</spanx> from
+                <spanx style="emph">filename</spanx>.</t>
+              <t>There is no limitation on the length of a pathname.</t>
+              <t>The payload manifest &must-not; reference files outside the payload directory.</t>
+              <t>
+                If a <spanx style="emph">filename</spanx> includes a newline
+                (LF), a carriage return (CR),
+                or carriage return plus newline (CRLF) it &must; be
+                percent-encoded following <xref target="RFC3986"/>.
+              </t>
+            </list>
+          </t>
+          <t>
+A manifest &must-not; reference directories. Bag creators who wish to create
+an otherwise empty directory have typically done so by creating an empty
+placeholder file with a name such as ".keep".
+</t>
+        </section>
+        <!-- /Payload Manifest -->
+      </section>
+      <!-- /Required Elements -->
+      <section title="Optional Elements" anchor="sec-optional-elements">
+        <section anchor="sec-tag-manifest" title="Tag Manifest: tagmanifest-algorithm.txt">
+          <t>
+            A tag manifest is a tag file that lists other tag files and checksums for
+            those tag files generated using a particular bag checksum algorithm.
+            A bag &may; contain one or more tag manifests.
+
+            A tag manifest file &must; have a name of the form "tagmanifest-<spanx style="emph">algorithm</spanx>.txt", where <spanx style="emph">algorithm</spanx>
+            is a string following the format described in
+            <xref target="bag-checksum-algorithms"/> specifying the bag checksum algorithm
+            used in that manifest.
+          </t>
+          <figure>
+            <preamble>Example tag manifest filenames:</preamble>
+            <artwork>
 tagmanifest-md5.txt
 tagmanifest-sha1.txt
- </artwork>
-</figure>
-
-<t>
+  </artwork>
+          </figure>
+          <t>
 A tag manifest file has the same form as the payload file manifest
```

Comment:
That was our intention and it's a good point for clarification

Reply:
I've added https://github.com/LibraryOfCongress/bagit-spec/pull/19 to clarify that intention - left it as a "SHOULD list all tag files" unless you agree it would be MUST.